zerver/tests/test_auth_backends.py

```diff
@@ -1027,9 +1031,12 @@ def test_social_auth_registration_existing_account(self) -> None:
     def stage_two_of_registration(self, result: HttpResponse, realm: Realm, subdomain: str,
                                   email: str, name: str, expected_final_name: str,
                                   skip_registration_form: bool,
+                                  mocked_social_avatar: Optional[BytesIO]=None,
```

Comment:
wouldn't it make more sense to give a path to the file with the content here, rather than tasking the tests with the hassle of making a `BytesIO` object? It would spare people writing new tests some trouble. Something like `expected_social_avatar_content_path` would perhaps be a better name too?